src/components/base/containers/Tooltip.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component, PropTypes } from 'react';
+import { findDOMNode } from 'react-dom';
+import Overlay from 'react-overlays/lib/Overlay';
+import Transition from 'react-overlays/lib/Transition';
+import styled, { injectGlobal } from 'styled-components';
+import { colors } from '../../theme';
+
+injectGlobal(`
```

Comment:
The transition component doesn't like that... it's not really rendering a wrapping element, it looks like it renders the children of the transition element and passes along the classes.